src/python_testing/TC_SEAR_1_5.py

```diff
@@ -89,7 +89,7 @@ async def send_cmd_skip_area_expect_response(self, step, skipped_area, expected_
                              f"Command response ({ret.status}) doesn't match the expected one")
 
     def TC_SEAR_1_5(self) -> list[str]:
-        return ["SEAR.S", "SEAR.S.C02.Rsp"]
+        return ["SEAR.S"]
```

Comment:
Same here - let's leave the top level PICS as-is for now and we'll deal with the top level PICS separately.

Reply:
This has been updated to leave the top level PICS as-is.